### PR TITLE
Github actions for essentails version

### PR DESCRIPTION
**Pull request by hageboeck**
- Make essentials reachable via makefile
- Add essentials to github builds

### CODE REVIEW

File: talk/C++Course.tex [hunk 1]
```diff
@@ -2,12 +2,11 @@
 \usetheme{Warsaw}
 \useoutertheme{split}
 
-%\includeonly{python/python}
-
 \input{setup}
 
 \newboolean{onlybasics}
 \setboolean{onlybasics}{false}
+\input{onlybasics}
```
Comment (bernhardmgruber) [inline talk/C++Course.tex:9]: Could you please test first, if the file exists? Something like:
```suggestion
\IfFileExists{onlybasics.tex}{\input{onlybasics}}{}
```
This way, I do not need to create the file myself if I use an IDE like `TexStudio` or an interface such as Overleaf.

**Comment by sponce** [comment]
I'm not liking too much using an extra file, but I must say that I fail finding a better solution in 15mn... there is this hack : https://tex.stackexchange.com/questions/1492/passing-parameters-to-a-document (second answer, not validated one) where you run : 
```
pdflatex "\setboolean{onlybasics}{true} \input{C++Course.tex}"
```
Although it probably does not even work with setboolean.

So I'm fine with what works :-)

**Review by sponce — APPROVED** [review]

**Comment by hageboeck** [comment]
> I'm not liking too much using an extra file, but I must say that I fail finding a better solution in 15mn... there is this hack : https://tex.stackexchange.com/questions/1492/passing-parameters-to-a-document (second answer, not validated one) where you run :
> 
> ```
> pdflatex "\setboolean{onlybasics}{true} \input{C++Course.tex}"
> ```
> 
> Although it probably does not even work with setboolean.
> 
> So I'm fine with what works :-)

@sponce, the only other ideas I had:
- Create a `C++CourseEssentials.tex`; add to repo
- Use sed to replace the boolean, but then git will tell you that you changed the file.

I'm happy to leave this undecided until Monday, maybe somebody comes up with a solution.

**Review by bernhardmgruber — APPROVED** [review]

**Review by klieret — APPROVED** [review]

**Comment by sponce** [comment]
"Perfect is the enemy of good", let's merge this.

**Pull request merged**